gui/app/models/graphql/GraphQlSchemaDefinition.scala

```diff
@@ -36,7 +36,9 @@ object GraphQlSchemaDefinition extends BaseGraphQlSchemaDefinition {
   val RootQueryType = ObjectType("RootQuery",  fields[GraphQlSchemaContext, Unit](
     Field("matchingNodes", ListType(NodeType), arguments = LimitArgument :: OffsetArgument :: TextArgument :: Nil, resolve = ctx => ctx.ctx.store.getMatchingNodes(limit = ctx.args.arg(LimitArgument), offset = ctx.args.arg(OffsetArgument), text = ctx.args.arg(TextArgument))),
     Field("matchingNodesCount", IntType, arguments = TextArgument :: Nil, resolve = ctx => ctx.ctx.store.getMatchingNodesCount(text = ctx.args.arg(TextArgument))),
-    Field("nodeById", OptionType(NodeType), arguments = IdArgument :: Nil, resolve = ctx => ctx.ctx.store.getNodeById(ctx.args.arg(IdArgument)))
+    Field("nodeById", OptionType(NodeType), arguments = IdArgument :: Nil, resolve = ctx => ctx.ctx.store.getNodeById(ctx.args.arg(IdArgument))),
+    Field("totalNodesCount", IntType, resolve = ctx => ctx.ctx.store.getTotalNodesCount()),
+    Field("totalEdgesCount", IntType, resolve = ctx => ctx.ctx.store.getTotalEdgesCount())
```

Comment:
Nit: please preserve alphabetical order when you're working on code I started.

Reply:
I fixed this.

Reply:
Ah should have picked up on this, will keep in mind